src/cosl/coordinated_workers/coordinator.py

```diff
@@ -613,6 +613,15 @@ def _on_collect_unit_status(self, e: ops.CollectStatusEvent):
         elif not self.s3_ready:
             statuses.append(ops.BlockedStatus("[s3] S3 not ready (probably misconfigured)."))
 
+        if self._external_url.startswith("https://") and not self.tls_available:
+            statuses.append(
+                ops.BlockedStatus(
+                    "[tls] misconfigured: the ingressed address is https, "
```

Comment:
nit: is there a reason why this message is `[tls] misconfigured` while the worker has `TLS misconfigured`? Should they match?

Reply:
the `[category] message` convention is only in the coordinator; worker statuses don't do that (which is something we may want to fix one day, but not here)